enterprise/backend/src/metabase_enterprise/sandbox/models/group_table_access_policy.clj

```diff
@@ -47,7 +47,7 @@
 
 (s/defn check-columns-match-table
   "Make sure the result metadata data columns for the Card associated with a GTAP match up with the columns in the Table
-  that's getting GTAPped. It's ok to remove columns, but you cannot add new columns. The base types of the Card
+  that's getting GTAPped. It's ok to remove columns, but added columns may be ignored. The base types of the Card
```

Comment:
should this be `added columns will be ignored`?

Reply:
Good catch